### PR TITLE
13035: set up filed eas route and associated templates

### PR DESCRIPTION
**Pull request by trbmcginnis**
**Big Picture Summary**
Create Filed EAS package route and associated templates

**Which major feature does this fit into?**
The Filed EAS Package

**Technical Explanation — How does it work?**
Create a route for filed-eas along with associated `edit` and `show` templates. Display the filed-eas package link button on the project page in the package list

Closes [AB#13035](https://dcp-paperless.visualstudio.com/d36fd830-9029-4b77-b0c4-b0df2012eb98/_workitems/edit/13035)

### CODE REVIEW

File: client/app/components/project/package-list-item.hbs [hunk 1]
```diff
@@ -110,8 +110,9 @@
     (if (eq @package.dcpPackagetype (optionset 'package' 'dcpPackagetype' 'code' 'DRAFT_EAS')) 'draft-eas.edit' 'draft-eas.edit') --}}
```
Comment (godfreyyeung) [inline client/app/components/project/package-list-item.hbs:110]: realized my comment here was wrong, it should have said ...'final-eas.edit' at the end 😅  
Thanks taylor! 
Can you also delete these (poorly written) comments?

Comment (trbmcginnis) [reply client/app/components/project/package-list-item.hbs:110]: Sure :)

File: client/app/components/packages/filed-eas/edit.hbs [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<div class="grid-x grid-margin-x">
+  <div class="cell large-8">
+
+    <section class="form-section">
+      <h1 class="header-large">
+        Environmental Assessment Statement (EAS)
```
Comment (godfreyyeung) [inline client/app/components/packages/filed-eas/edit.hbs:6]: This is completely my bad but can you add "Filed" in front of line 6 here? And also help me add "Draft" to the draft edit component? Sorry and thank you!!

File: client/app/routes/filed-eas.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import Route from '@ember/routing/route';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+
+export default class FiledEasRoute extends Route.extend(AuthenticatedRouteMixin) {
+  authenticationRoute = '/';
+
+  async model(params) {
+    const FiledEasPackage = await this.store.findRecord('package', params.id, {
```
Comment (godfreyyeung) [inline client/app/routes/filed-eas.js:8]: nit: this should be undercase `filedEasPackage`

**Review by godfreyyeung — APPROVED** [review]

**Pull request merged**